library/core/src/cell/lazy.rs

```diff
@@ -55,6 +55,29 @@ impl<T, F> LazyCell<T, F> {
     pub const fn new(init: F) -> LazyCell<T, F> {
         LazyCell { cell: OnceCell::new(), init: Cell::new(Some(init)) }
     }
+
+    /// Consumes the lazy, returning the evaluated value.
+    ///
+    /// Returns `None` if the lazy was never evaluated.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(once_cell)]
+    ///
+    /// use std::cell::LazyCell;
+    ///
+    /// let hello = "Hello, World!".to_string();
+    ///
+    /// let lazy = LazyCell::new(|| hello.to_uppercase());
+    ///
+    /// assert_eq!(&*lazy, "HELLO, WORLD!");
+    /// assert_eq!(lazy.into_inner(), Some("HELLO, WORLD!".to_string()));
+    /// ```
+    #[unstable(feature = "once_cell", issue = "74465")]
+    pub fn into_inner(self) -> Option<T> {
```

Comment:
As `LazyCell` implements `Deref`, this should be an associated method to avoid conflicts with the methods of the stored type.

Reply:
Done.